### DISCUSSION TITLE
想问一下是不是一天不用就需要重新运行？

### DISCUSSION THREAD

**Discussion by attack-flower** [zhayujie/chatgpt-on-wechat#806] [Q&A]
我的机器人只有我自己用。发现一天不用，再发信息就不回复了。需要上服务器重新运行扫码才行。。是有有效期吗？
另外想问一下可以接入其他ai语音的api？有现成的代码替换吗？

**Comment by zhenda-hub** [comment]
突然就开始重启了。[INFO][2023-05-05 07:20:32][chat_channel.py:53] - context.get("isgroup"): True
[INFO][2023-05-05 07:20:32][chat_channel.py:60] - at here!!!
[INFO][2023-05-05 07:20:39][chat_channel.py:53] - context.get("isgroup"): True
[INFO][2023-05-05 07:20:39][chat_channel.py:60] - at here!!!
[INFO][2023-05-05 07:20:44][config.py:166] - 配置文件不存在，将使用config-template.json模板
[INFO][2023-05-05 07:20:44][config.py:170] - [INIT] config str: {
"open_ai_api_key": "xxx",
"model": "gpt-3.5-turbo",
"proxy": "",
"single_chat_prefix": [
"bot",
"@bot"
],
"single_chat_reply_prefix": "[bot] ",
"group_chat_prefix": [
"@bot"
],
"group_name_white_list": [
"ALL_GROUP"
],
"group_chat_in_one_session": [
"ChatGPT测试群"
],
"image_create_prefix": [
"画",
"看",
"找"
],
"image_create_size": "512x512",
"debug": 0,
"speech_recognition": false,
"group_speech_recognition": false,
"voice_reply_voice": false,
"conversation_max_tokens": 1000,
"expires_in_seconds": 3600,
"character_desc": "你是ChatGPT, 一个由OpenAI训练的大型语言模型, 你旨在回答并解决人们的任何问题，并且可以使用多种语言与人交流。",
